### DISCUSSION TITLE
Concerning linkage to external css file in Play CDN

### DISCUSSION THREAD

**Discussion by Scaramask** [tailwindlabs/tailwindcss#8617] [Show and tell]
Hello Friends, hope you doing fine 👍🏽

I am doing a test with Tailwind Play CDN and currently I already host externaly my tailwind_config.js and link to it with success full from my index.html. Afterwards I also tried to host externaly my taiwind.css file in my architecture and link to it from my index.html but without success unfortunately. Can someone give me an example.

You can check my code on my Codepen.

https://codepen.io/Scriptintest/pen/vYdvJyG

Thank you for your help

**Comment by dtydev** [comment]
I have the same problem, is there a solution?

_It seems that it is impossible to do..._

**Comment by Carlsia** [comment]
Glad you like it. We have been using this for a colossal piece of a time now and it is clearly going charmingly.
[AIM Specialty Health Provider Portal](https://www.aimproviderportal.org/)
